blog/tests/test_views.py

```diff
@@ -0,0 +1,108 @@
+"""Test for views."""
+from datetime import datetime
+from unittest.mock import patch
+
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.urls import reverse
+
+from ..models import Post
+
+
+class PostViewTest(TestCase):
+    """Test for post views."""
+
+    USERNAME = 'testuser'
+    PASSWORD = 'password'
+    EMAIL = 'email@test.com'
+
+    def setUp(self):
+        """Pre-populate test data."""
+        self.client = Client()
+        self.user = User.objects.create_superuser(username=self.USERNAME,
+                                                  password=self.PASSWORD,
+                                                  email=self.EMAIL,)
```

Comment:
put the closing paren on the next line: trailing commas are there to allow cleaner diffs, if you put parentheses there it becomes useless.